src/help/components/entry/styles/entry-preview.module.scss

```diff
@@ -0,0 +1,28 @@
+@import 'franklin-sites/src/styles/settings';
+
+.private-area {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  inline-size: 100%;
+  min-block-size: 100px;
+  padding: $global-padding;
+  background: lightcoral;
```

Comment:
Was a little tricky to read the text with the contrast, how about a slightly lighter tone such as `peachpuff` or `moccasin`?

Reply:
Changed in https://github.com/ebi-uniprot/uniprot-website/pull/539/commits/77e306f69c6cc29745cbbbf70e826b1e14908ce8